config/routes.rb

```diff
@@ -274,11 +274,16 @@
     end
   end
 
+  resources :global_user_roles do
+    collection do
```

Comment:
This can also be done on one line as `get "search", on: :collection`. Not a big deal, though.